doc/source/dask-on-ray.rst

```diff
@@ -25,50 +25,73 @@ Here's an example:
 
    import ray
    from ray.util.dask import ray_dask_get
-   import dask.delayed
+   import dask.array as da
+   import dask.dataframe as dd
+   import numpy as np
+   import pandas as pd
    import time
 
    # Start Ray.
    # Tip: If you're connecting to an existing cluster, use ray.init(address="auto").
    ray.init()
 
+   d_arr = da.from_array(np.random.randint(0, 1000, size=(256, 256)))
 
-   @dask.delayed
-   def inc(x):
-       time.sleep(1)
-       return x + 1
+   # The Dask scheduler submits the underlying task graph to Ray.
+   d_arr.mean().compute(scheduler=ray_dask_get)
 
-   @dask.delayed
-   def add(x, y):
-       time.sleep(3)
-       return x + y
+   # Set the scheduler to ray_dask_get in your config so you don't have to specify it on
+   # each compute call.
+   dask.config.set(scheduler=ray_dask_get)
+
+   df = dd.from_pandas(pd.DataFrame(np.random.randint(0, 100, size=(1024, 2)), columns=["age", "grade"]))
+   df.groupby(["age"]).mean().compute()
 
-   x = inc(1)
-   y = inc(2)
-   z = add(x, y)
-   # The Dask scheduler submits the underlying task graph to Ray.
-   z.compute(scheduler=ray_dask_get)
 
 Why use Dask on Ray?
 
-   1. If you'd like to create data analyses using the familiar NumPy and Pandas
-      APIs provided by Dask and execute them on a production-ready distributed
-      task execution system like Ray.
-   2. If you'd like to use Dask and Ray libraries in the same application
-      without having two different task execution backends.
+   1. If you'd like to create data analyses using the familiar NumPy and Pandas APIs provided by Dask and execute them on a fast, fault-tolerant distributed task execution system geared towards production, like Ray.
+   2. If you'd like to use Dask and Ray libraries in the same application without having two different clusters.
    3. To take advantage of Ray-specific features such as the
       :ref:`cluster launcher <ref-automatic-cluster>` and
       :ref:`shared-memory store <memory>`.
 
 Note that for execution on a Ray cluster, you should *not* use the
 `Dask.distributed <https://distributed.dask.org/en/latest/quickstart.html>`__
 client; simply use plain Dask and its collections, and pass ``ray_dask_get``
```

Comment:
Please explain this is "currently" not supported. (We can probably starting supporting it once ray client becomes more stable)

Reply:
I don't think that we will ever want to implement Dask Distributed's scheduler protocol and/or the client API, at least not in its current form. I think that it's more likely that we would offer our own client that offers an API that better fits the task/cluster state introspection and management capabilities and APIs that Ray exposes.